pkg/ip/ip.go

```diff
@@ -85,31 +85,97 @@ func IsIP(version types.IPVersion, s string) error {
 //
 // If sorted is true, the result set of IP addresses will be sorted.
 func IPsDiffSet(ips1, ips2 []net.IP, sorted bool) []net.IP {
-	var ips []net.IP
-	marks := make(map[string]bool)
-	for _, ip := range ips1 {
+	return getIPDiffSet(ips1, ips2, sorted, -1)
+}
+
+func IsDiffIPSet(ips1, ips2 []net.IP) bool {
+	diff := getIPDiffSet(ips1, ips2, false, 1)
+	return len(diff) > 0
+}
+
+func getIPDiffSet(ips1, ips2 []net.IP, sorted bool, want int) []net.IP {
```

Comment:
add some comments for the parameters
ips1 -> ipSourcelist
ips2 -> ipExcludeList
want -> expectCount
sorted -> sorted